src/Decompiler/Analysis/SsaTransform.cs

```diff
@@ -1264,7 +1270,7 @@ private static void GeneratePhiFunction(SsaIdentifier phi, SsaIdentifier[] sids)
             /// </summary>
             /// <param name="phi"></param>
             /// <returns></returns>
-            private SsaIdentifier TryRemoveTrivial(SsaIdentifier phi)
```

Comment:
Consider documenting this function, explaining what returning true and false mean.

Reply:
I have done rebase to add documentation (see https://github.com/uxmal/reko/pull/638#discussion_r191077306).